js_modules/dagit/packages/core/src/graph/SVGViewport.tsx

```diff
@@ -438,6 +450,30 @@ export class SVGViewport extends React.Component<SVGViewportProps, SVGViewportSt
     this.props.onDoubleClick && this.props.onDoubleClick(event);
   };
 
+  onExportToSVG = () => {
+    const unclippedViewport = {
+      top: 0,
+      left: 0,
+      right: this.props.graphWidth,
+      bottom: this.props.graphHeight,
+    };
+
+    const div = document.createElement('div');
+    document.getElementById('root')!.appendChild(div);
+    ReactDOM.render(
+      <MemoryRouter>{this.props.children(this.state, unclippedViewport)}</MemoryRouter>,
+      div,
+    );
+    const svg = div.querySelector('svg') as SVGElement;
+    makeSVGPortable(svg);
+
+    const text = new XMLSerializer().serializeToString(svg);
+    const blob = new Blob([text], {type: 'image/svg+xml'});
+    const url = URL.createObjectURL(blob);
+    open(url);
+    div.remove();
+  };
```

Comment:
Ahh interesting that's true - I think this should be ok because we attach and then immediately remove the div -- it just needs to be mounted briefly so that the page's CSS styles apply to the node and we can inspect them.